text/machine_comprehension/roberta/README.md

```diff
@@ -11,23 +11,23 @@ RoBERTa builds on BERT’s language masking strategy and modifies key hyperparam
  |Model        |Download  |Download (with sample test data)| ONNX version |Opset version|Accuracy|
 | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
 |RoBERTa-BASE| [499 MB](model/roberta-base-11.onnx) |  [295 MB](model/roberta-base-11.tar.gz) |  1.6 | 11|[96.7](https://paperswithcode.com/paper/roberta-a-robustly-optimized-bert-pretraining/review/?hl=6023)|
-|RoBERTa-STT-2| [501 MB](model/roberta-stt-9.onnx) |  [434 MB](model/roberta-stt-9.tar.gz) |  1.4 | 9|[88.5](https://paperswithcode.com/paper/roberta-a-robustly-optimized-bert-pretraining/review/?hl=6023)|
+|RoBERTa-SequenceClassification| [501 MB](model/roberta-stt-9.onnx) |  [434 MB](model/roberta-stt-9.tar.gz) |  1.4 | 9|[88.5](https://paperswithcode.com/paper/roberta-a-robustly-optimized-bert-pretraining/review/?hl=6023)|
```

Comment:
For accuracy 88.5, which data set do you mean? 

From the notebook URL, it seems the model was fine-tuned for movie sentiment classification. A search in SeldonIO [https://github.com/SeldonIO/alibi/blob/5aec3ab4ce651ca2249bf849ecb434371c9278e4/alibi/datasets.py] shows  movie sentiment data is http://www.cs.cornell.edu/People/pabo/movie-review-data/rt-polaritydata.tar.gz. It looks not same as STT dataset.

Reply:
88.5 is supposed to the average accuracy score of the roberta-base as it is a bare model with no head. It is given in the paper of the model. 

For the sequence classification model added the MCC score along with the validation notebook using the right dataset.

Reply:
I saw 88.5 is on the row of sequence classification model. It shall be in the row of roberta-base according to your comment. Right?

Reply:
It is the same row but includes the average of accuracies of roberta-base measured over multiple datasets. 
![Image 7-30-20 at 6 19 PM](https://user-images.githubusercontent.com/49421967/88989776-4355aa80-d291-11ea-9540-1e22e2b6261f.jpg)